scripts/sct_label_utils.py

```diff
@@ -296,7 +297,9 @@ def main(args=None):
         else:
             out = launch_sagittal_viewer(img, arguments.create_viewer, msg)
 
+    logger.info("Generating output files...")
```

Comment:
are we supposed to use the logger in the CLI? (i thought we decided to only use logger for the API)

Reply:
Good question. You're right. I should be using `printv` here, as per the [dev Wiki](https://github.com/neuropoly/spinalcordtoolbox/wiki/Programming%3A-Logging).